alfresco-search/src/main/java/org/alfresco/solr/SolrInformationServer.java

```diff
@@ -2656,7 +2656,7 @@ public void indexTransaction(Transaction info, boolean overwrite) throws IOExcep
 
     private void indexNonShardCascade(NodeMetaData nodeMetaData) throws IOException
     {
-        canUpdate();
+
```

Comment:
So it was a matter of thread registration?

Reply:
Yes. The indexNonShardCascade is called by an asynchronous worker (not registered) started in metadata tracker. The canUpdate() method is not necessary here and does not affect the correctness of the execution.